scripts/ci-test-lane.sh

```
@@ -0,0 +1,358 @@
+if [ "$status1" -eq 0 ]; then
+  extract_bundle "$bundle1"
+  finish_lane "pass" '[{"n": 1, "mode": "lane", "status": "passed"}]' "" 0
+fi
+
+extract_bundle "$bundle1"
+FAIL_COUNT=$(count_failures)
+
+# Ordinary test failures: never rerun the healthy lane. Native retry already
+# re-ran only the failing tests; survivors are real failures.
+if [ "$status1" -ne 124 ] && [ "$FAIL_COUNT" -gt 0 ]; then
+  echo "lane $LANE: "${FAIL_COUNT}" test(s) failed after native retry"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)